ui/dashboards/src/components/AddPanelButton/AddPanelButton.stories.tsx

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import type { Meta, StoryObj } from '@storybook/react';
+import { AddPanelButton } from '@perses-dev/dashboards';
+import {
+  WithTemplateVariables,
+  WithQueryParams,
+  WithDashboard,
+  WithPluginRegistry,
+  WithQueryClient,
+} from '../../stories/decorators';
+
+const meta: Meta<typeof AddPanelButton> = {
+  component: AddPanelButton,
```

Comment:
i think we might need to put a wrapper here... the tooltip is misplaced when you hover over the button

Reply:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/9817826/224167872-901e10fc-7bd5-4a9f-991f-7b4d02f4b1b4.png">

Reply:
I can't seem to reproduce this trying out a handful of the stories and the perses app in chrome, firefox, or safari. Wondering if I'm missing an edge case. Could you let me know where you saw this and using which browser? E.g. was it in the app or one of the storybook. If the latter which story and if it was in docs mode or the direct story.

Reply:
Can't reproduce it anymore after pulling the newest changes, i'll mark it as resolved!

Reply:
FYI for others, that Eunice couldn't reproduce this. Guessing she hit it with an outdated version of the code before I fixed the tooltip.